rfcs/0130-stalled-rfcs.md

```diff
@@ -0,0 +1,145 @@
+
+The "Lacking Interest" state defines an RFC that can not gather a shepherd team. This is taken to indicate that there is not enough interest in the Nix community. These RFCs will be Closed until more interest is gathered.
+
+# Motivation
+[motivation]: #motivation
+
+The NixOS RFC process describes a process with a focus on moving RFCs toward completion. This is where an engaged author and community push an RFC to a decision, either to accept or reject the proposal.
+
+However things don't always work like this. Sometimes the author doesn't have sufficent time or motivation to keep iterating on the RFC and sometimes the community doesn't have enuogh interested members to staff a shepherd team.
+
+It is important that these "alternate exits" are well defined to avoid wasting effort trying to push a stalled RFC forward and to set clear expectations for all participants.
+
+TL;DR timeouts are critical in distributed systems.
+
+# Detailed design
+[design]: #detailed-design
+
+Here is an updated diagram for the RFC process.
+
+```mermaid
+graph TD
+    Start((Start)) --> Draft
+    Draft -->|Ready for Review| Discuss
+    style Draft fill:#008,color:#FFF
+
+    Discuss[Discuss and Refine]
+    Discuss --> |On Hold| Draft
+    Discuss --> |Can't Find Shepherds| NoShepherds
+    Discuss --> |Motion for FCP| FCP
+
+    NoShepherds[Closed - Lack of Interest]:::closed
+    NoShepherds --> |Renewed Interest| Discuss
+
+    FCP[Final Coment Phase]
+    FCP --> |FCP Canceled| Discuss
+    FCP --> |Accept| Merged
+    FCP --> |Reject| Rejected
+    
+    Merged
+    style Merged fill:#080,color:#FFF
+    
+    Rejected[Closed - Rejected]:::closed
+
+    Withdrawn[Closed - Withdrawn]:::closed
+    Discuss & Draft --->|Author Withdraws| Withdrawn
+
+    classDef closed fill:#800,color:#FFF
+```
+
+There are 3 new transitions in this diagram.
+
+## On Hold
+
+This is for cases where the author is unable or unwilling to update the RFC in a timely fashon and would like to indicate that the RFC should not continue to be pushed forward for the time being.
+
+The author can [mark the PR as a Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft) and the existing label [status: on hold](https://github.com/NixOS/rfcs/pulls?q=is%3Apr+label%3A%22status%3A+on+hold%22+is%3Aclosed) can be applied to the PR by the NixOS RFC Steering Committee for tracking reasons.
+
+RFCs in this status will be completely ignored by the NixOS RFC Steering Committee. It is also expected that community members will largely refrain from review until the author has indicated that the RFC is again ready for review.
+
+## Can't Find Shepherds
+
+If an RFC is unable to find adaqite number of shepherds in 2 months it will be considered "low interest". The RFC will be closed to indicate that there is no expection of continuous forward progress.
```

Comment:
```suggestion
If an RFC is unable to find adequate number of shepherds in 2 months it will be considered "low interest". The RFC will be closed to indicate that there is no expectation of continuous forward progress.
```